### ISSUE TITLE
Transparent bg on image preview

### ISSUE THREAD

**Issue by zanhk** [sveltia/sveltia-cms#76]
### Sveltia info

Sveltia 0.8.4

### Problem

If I upload a white path png on the collection list it appear with a white background, 

![image](https://github.com/sveltia/sveltia-cms/assets/17980844/e39dfd3d-679b-4359-9dfb-9d60043d36ca)

on the editor instead I see the image correctly, but not on the preview

![image](https://github.com/sveltia/sveltia-cms/assets/17980844/230ed357-d708-4b14-8329-6ed3a9c9dc78)

### What's the expected result?

All images positions should have the same background:

```
background-image: linear-gradient(45deg, #eee 25%, transparent 25%, transparent 75%, #eee 75%), linear-gradient(45deg, #eee 25%, #fff 25%, #fff 75%, #eee 75%);
background-size: 16px 16px;
background-position: 0 0, 8px 8px;
```

![image](https://github.com/sveltia/sveltia-cms/assets/17980844/7b6d7a60-2f36-4646-accc-0231285b1614)

### Participation

- [x] I am willing to submit a pull request for this issue.

**Comment by kyoshino** [comment]
Will check what’s happening here.

**Comment by kyoshino** [comment]
Just shipped [v0.9.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.9.0) with the fix! 🎉
